tools/e2e-commons/bin/e2e-env.sh

```diff
@@ -54,6 +54,8 @@ gb_setup() {
 }
 
 configure_wp_env() {
+	$BASE_CMD wp plugin status
```

Comment:
Do we need these?

Reply:
Yeah, I left them there intentionally, to check the state of the plugins. I think is helpful at least for now. We can remove them later if they get annoying.